homeassistant/components/cloud/__init__.py

```diff
@@ -60,17 +66,19 @@ def async_setup(hass, config):
         kwargs = {CONF_MODE: DEFAULT_MODE}
 
     if CONF_ALEXA not in kwargs:
-        kwargs[CONF_ALEXA] = ALEXA_SCHEMA({})
+        kwargs[CONF_ALEXA] = ASSISTANT_SCHEMA({})
 
-    kwargs[CONF_ALEXA] = smart_home.Config(**kwargs[CONF_ALEXA])
+    if CONF_GOOGLE_ASSISTANT not in kwargs:
+        kwargs[CONF_GOOGLE_ASSISTANT] = ASSISTANT_SCHEMA({})
+
+    kwargs[CONF_ALEXA] = alexa_sh.Config(**kwargs[CONF_ALEXA])
+    kwargs['gass_should_expose'] = kwargs.pop(CONF_GOOGLE_ASSISTANT)
     cloud = hass.data[DOMAIN] = Cloud(hass, **kwargs)
 
-    @asyncio.coroutine
-    def init_cloud(event):
-        """Initialize connection."""
-        yield from cloud.initialize()
+    success = yield from cloud.initialize()
```

Comment:
```
if not (yield from ...):
    return False
```

Reply:
I am not a fan of inline yields

Reply:
I hope you change your mind with `if await cloud.init():` :)